index.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
```

Comment:
I would suggest not to have this file in the root directory. This can probably go into a folder like config or something.

Reply:
sure, i'll have to create the folder though

Reply:
yes